Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ruby:3.1.2
+FROM ruby:3.2.2
```

Comment:
Can we update the `.dockerignore` file here as well? 🙏

Reply:
maybe this should be done in a separate PR. I don't think this one will be merged for quite some time